src/Simplex/Chat/Controller.hs

```diff
@@ -201,7 +226,20 @@ instance ToJSON HelpSection where
   toEncoding = J.genericToEncoding . enumJSON $ dropPrefix "HS"
 
 data ChatCommand
-  = ShowActiveUser
+  = CreateRemoteHost -- ^ Configure a new remote host
+  | ListRemoteHosts
+  | StartRemoteHost RemoteHostId -- ^ Start and announce a remote host
+  | StopRemoteHost RemoteHostId -- ^ Shut down a running session
+  | DisposeRemoteHost RemoteHostId -- ^ Unregister remote host and remove its data
+  | StoreRemote FilePath RemoteHostId -- ^ Store a file at host, preparing for `SendFile`
+  | FetchRemote FilePath RemoteHostId -- ^ Fetch a file from host after `ReceiveFile`
```

Comment:
it's arguable whether these should be commands or just functions... We will see, ok to keep as commands for now.

Reply:
The reason why they should not be commands is because it would offload this responsibility to the UI (and we are back to doing it 4 times), while when desktop core receives send from UI it can automatically upload file to host before forwarding the actual send command, and likewise having received "file received" it may also automatically fetch it. Also, not sure why the actual file cannot be just attached as a body to the request or response and any commands are needed here at all.

Kind of convinced myself these commands are not needed, normally. The only case they may useful is when you load chat, and there is some file, and it's not present in the local storage. But again, the host could simply see it before sending the response, check if it has files locally, and send them with the response.